src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -271,11 +271,11 @@
         <summary>Gets the type of the tuple element.</summary>
       </PropertyComment>
     </Field>
-    <Field Name="Name" Type="IdentifierNameSyntax" Optional="true">
+    <Field Name="Name" Type="SyntaxToken" Optional="true">
```

Comment:
This field should probably be renamed `Identifier` to be consistent with convention.